kubernetes/start

```diff
@@ -29,7 +30,7 @@ while (( $interactive_mode == 1)) ; do
   esac
 done
 
-./cleanup
+./cleanup || true
```

Comment:
Maybe? The cleanup script needs some work then. It will currently fail if there are no containers to clean up.